__tests__/fixture/lends.input.js

```diff
@@ -6,6 +6,8 @@
 export default TheClass(
   /** @lends TheClass.prototype */
   {
+    /** My field */
+    myField: true,
```

Comment:
prettier or something similar is changing this line from `"myField"` into `myField`, ... looking for proper ignore annotation